.pre-commit-config.yaml

```diff
@@ -49,6 +49,7 @@ repos:
   rev: 6.3.0
   hooks:
     - id: pydocstyle
+      additional_dependencies: [tomli]
```

Comment:
what's this for?

Reply:
When I ran the pre-commit hooks the pydocstyle was failing for me, complaining that `tomli` was not installed...

Reply:
Might be better to leave out of this particular PR then... Maybe it would be better raised as an issue?  I just don't know much about precommit and these things and so would rather not slip this in with a GP PR.

Reply:
Will remove